lib/vquic/curl_ngtcp2.c

```diff
@@ -2190,9 +2192,12 @@ static void cf_ngtcp2_close(struct Curl_cfilter *cf, struct Curl_easy *data)
 static void cf_ngtcp2_destroy(struct Curl_cfilter *cf, struct Curl_easy *data)
 {
   struct cf_ngtcp2_ctx *ctx = cf->ctx;
+  /*
   struct cf_call_data save;
 
   CF_DATA_SAVE(save, cf, data);
+  */
+  (void)data;
```

Comment:
I think it would be safer to do it as (void)save, unless @icing is ok with it as is. The CF_DATA_SAVE writes the current easy handle to call_data which is possibly used by the other cleanup functions.